examples/Python/src/CARLA/carla_circle.lf

```diff
@@ -0,0 +1,242 @@
+target Python {
+  threading: True
+}
+
+preamble {=
+  import carla

```

Comment:
Nit: this big chunk of Python code might be clearer and easier to work with if it were factored out into a separate `.py` file, but this is not important and addressing it is totally optional